WordPress/src/main/res/values/strings.xml

```diff
@@ -955,6 +955,7 @@
     <string name="invalid_password_message">Password must contain at least 4 characters</string>
     <string name="invalid_username_too_short">Username must be longer than 4 characters</string>
     <string name="invalid_username_too_long">Username must be shorter than 61 characters</string>
+    <string name="invalid_username_no_spaces">Username cannot contain spaces</string>
```

Comment:
`s/cannot/can't/` - See "Use contractions" section in https://www.google.com/design/spec/style/writing.html#writing-capitalization-punctuation